app/views/sign_up/recovery_codes/show.html.slim

```diff
@@ -16,5 +16,14 @@ p.mt-tiny.mb0 = t('instructions.recovery_code')
     data: { print: true },
     class: 'ml2 radio-btn inline-block ico ico-print text-decoration-none'
 
+.mb4.col-12.border.br-md.border-blue.fs-16p.accordion(aria-expanded='true')
+  .accordion-header.py1.px2(aria-controls='recovery-code-info')
```

Comment:
On smaller mobile screens the + icon overlaps the link text.  I think if we swap out `px2` for `pr3 pl2` that should give the right side some extra padding just in case.